list/list.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>List</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="../home/styles.css" />
+    <link rel="stylesheet" href="./styles2-lists.css" />
+  </head>
```

Comment:
just a suggestion, you can enter your site name plus the page name like girlyfy| list in the title

Reply:
done